plugins/api/src/register_physical_path.cpp

```diff
@@ -21,56 +21,59 @@
 #include "irods/irods_at_scope_exit.hpp"
 #include "irods/irods_get_l1desc.hpp"
 #include "irods/irods_hierarchy_parser.hpp"
-#include "irods/irods_resource_backport.hpp"
-#include "irods/irods_resource_redirect.hpp"
-#include "irods/key_value_proxy.hpp"
-#include "irods/miscServerFunct.hpp"
-#include "irods/objMetaOpr.hpp"
-#include "irods/phyPathReg.h"
-#include "irods/physPath.hpp"
-#include "irods/rcGlobalExtern.h"
-#include "irods/rcMisc.h"
-#include "irods/resource.hpp"
-#include "irods/rodsLog.h"
-#include "irods/rsCollCreate.hpp"
-#include "irods/rsDataObjClose.hpp"
-#include "irods/rsDataObjCreate.hpp"
-#include "irods/rsFileClose.hpp"
-#include "irods/rsFileClosedir.hpp"
-#include "irods/rsFileMkdir.hpp"
-#include "irods/rsFileOpen.hpp"
-#include "irods/rsFileOpendir.hpp"
-#include "irods/rsFileRead.hpp"
-#include "irods/rsFileReaddir.hpp"
-#include "irods/rsFileStat.hpp"
-#include "irods/rsGlobalExtern.hpp"
-#include "irods/rsModAccessControl.hpp"
-#include "irods/rsModColl.hpp"
-#include "irods/rsRegColl.hpp"
-#include "irods/rsRegDataObj.hpp"
-#include "irods/rsRegReplica.hpp"
-#include "irods/rsSubStructFileStat.hpp"
-#include "irods/rsSyncMountedColl.hpp"
-#include "irods/server_utilities.hpp"
-#include "irods/specColl.hpp"
-#include "irods/stringOpr.h"
-
-#define IRODS_FILESYSTEM_ENABLE_SERVER_SIDE_API
-#include "irods/filesystem.hpp"
-
-#define IRODS_REPLICA_ENABLE_SERVER_SIDE_API
-#include "irods/replica_proxy.hpp"
-
-#include <boost/lexical_cast.hpp>
-#include <fmt/format.h>
-
-#include <iostream>
+#  include "irods/irods_logger.hpp"
+#  include "irods/irods_resource_backport.hpp"
+#  include "irods/irods_resource_redirect.hpp"
```

Comment:
Let's revert the formatting on all these include directives.

Right now, they look weird in relation to the include directives that weren't touched.